packages/shopping/src/__tests__/acceptance/user.controller.acceptance.ts

```diff
@@ -1,331 +1,331 @@
-        .set('Authorization', 'NotB3@r3r ' + 'xxx.yyy.zzz')
-        .expect(401);
-
-      expect(res.body.error.message).to.equal(
-        "Authorization header is not of type 'Bearer'.",
-      );
-    });
-
-    it('users/me returns an error when an expired JWT token is provided', async () => {
-      const res = await client
-        .get('/users/me')
-        .set('Authorization', 'Bearer ' + expiredToken)
-        .expect(401);
-
-      expect(res.body.error.message).to.equal(
-        'Error verifying token : jwt expired',
-      );
-    });
-  });
-
-  describe('user product recommendation (service) api', () => {
-    let recommendationService: HttpServer;
-    let recommendationGRPCService: Server;
-
-    before(async () => {
-      recommendationService = createRecommendationServer();
-      await recommendationService.start();
-      recommendationGRPCService = createGRPCRecommendationServer();
-      recommendationGRPCService.start();
-    });
-
-    after(async () => {
-      await recommendationService.stop();
-      recommendationGRPCService.forceShutdown();
-    });
-
-    it('returns product recommendations for a user', async () => {
-      const newUser = await createAUser();
-      await client
-        .get(`/users/${newUser.id}/recommend`)
-        .expect(200, recommendations['user001']);
-    });
-
-    it('returns product recommendations for a user using gRPC', async () => {
-      const recommender = await app.get<RecommenderService>(
-        'services.RecommenderGrpcService',
-      );
-      const products = await recommender.getProductRecommendations('user001');
-      expect(products).to.eql(recommendations['user001']);
-    });
-  });
-
-  async function clearDatabase() {
-    await userRepo.deleteAll();
-  }
-
-  async function migrateSchema() {
-    await app.migrateSchema();
-  }
-
-  async function createAUser() {
-    const encryptedPassword = await passwordHasher.hashPassword(userPassword);
-    const newUser = await userRepo.create(userData);
-    // MongoDB returns an id object we need to convert to string
-    newUser.id = newUser.id.toString();
-
-    await userRepo.userCredentials(newUser.id).create({
-      password: encryptedPassword,
-    });
-
-    return newUser;
-  }
-
-  async function createPasswordHasher() {
-    passwordHasher = await app.get(PasswordHasherBindings.PASSWORD_HASHER);
-  }
-
-  /**
-   * Creates an expired token
-   *
-   * Specifying a negative value for 'expiresIn' so the
-   * token is automatically expired
-   */
-  async function givenAnExpiredToken() {
-    const newUser = await createAUser();
-    const tokenService: JWTService = new JWTService(
-      TokenServiceConstants.TOKEN_SECRET_VALUE,
-      '-1',
-    );
-    const userProfile = {
-      [securityId]: newUser.id,
-      name: `${newUser.firstName} ${newUser.lastName}`,
-    };
-    expiredToken = await tokenService.generateToken(userProfile);
-  }
-});
+// // Copyright IBM Corp. 2018,2019. All Rights Reserved.
```

Comment:
Why is this acceptance test commented out?

Reply:
There was a build issue. Uncommented.